x-pack/plugins/maps/public/plugin.ts

```diff
@@ -158,8 +158,8 @@ export class MapsPlugin
     });
   }
 
-  public start(core: CoreStart, plugins: MapsPluginStartDependencies): MapsStartApi {
-    setLicensingPluginStart(plugins.licensing);
+  public async start(core: CoreStart, plugins: MapsPluginStartDependencies): Promise<MapsStartApi> {
```

Comment:
I thought async was getting removed from start contract? 

@joshdover can you add some context about plugin methods and async signatures?

Reply:
It is getting removed, probably at 8.0. Motivation is here: https://github.com/elastic/kibana/blob/master/rfcs/text/0007_lifecycle_unblocked.md

Tracking issue: https://github.com/elastic/kibana/issues/53268